### ISSUE TITLE
Bronchial diverticula

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6627]
**Preferred term label:**  Bronchial diverticula

**Synonyms**

**Definition (free text, please give PubMed ID)**
Bronchial diverticula are blind-ended outpouchings arising from the bronchial tree. They are commonly pulsion diverticula acquired related to chronic cough. Subcarinal air cyst is thought to represent a small main bronchial diverticulum. PMID: 18544671, PMID: 27867581

**Parent term (use hpo.jax.org/app)**
Abnormal bronchus morphology HP:0025426

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
